naptime-graphql/src/main/scala/org/coursera/naptime/ari/graphql/resolvers/NaptimeResolver.scala

```diff
@@ -185,15 +177,34 @@ class NaptimeResolver extends DeferredResolver[SangriaGraphQlContext] with Stric
     requests
       .groupBy(_.arguments.filterNot(_._1 == "ids"))
       .map { case (nonIdArguments, innerRequests) =>
-        val ids = innerRequests.flatMap(_.arguments.find(_._1 == "ids").map(_._2)).distinct
         // TODO(bryan): Limit multiget requests by number of ids as well, to avoid http limits
         Request(
           header,
           resourceName,
-          nonIdArguments + ("ids" -> JsArray(ids))) -> innerRequests
+          nonIdArguments + ("ids" -> JsArray(parseAndDeduplicateIds(innerRequests)))) -> innerRequests
       }
   }
 
+  private[this] def parseAndDeduplicateIds(innerRequests: Vector[NaptimeRequest]): Vector[JsValue] = {
+    val parsedIds = for {
+      request <- innerRequests
+      id <- parseAndDeduplicateIds(request)
+    } yield id
+    parsedIds.distinct
+  }
+
+  private[this] def parseAndDeduplicateIds(request: NaptimeRequest): Seq[JsValue] = {
+    request.arguments
+      .find(_._1 == "ids")
```

Comment:
I know this was like this before, but can we not use `_._1`, `_._2` far away from the definition location? Here `case (key, _)` and next line `.values` would be a lot more meaningful